pkg/driver/controllerserver.go

```diff
@@ -479,6 +518,10 @@ func parseCustomSecret(secret *v1.Secret) map[string]string {
 		locationConstraint = string(bytesVal)
 	}
 
+	if bytesVal, ok := secret.Data["BucketVersioning"]; ok {
```

Comment:
`bucketVersioning` -> small case in the start to comply with how other params are set in the secret.